### ISSUE TITLE
Mitm disable internet on devices

### ISSUE THREAD

**Issue by Palmus84** [Ettercap/ettercap#944]
Every time I use Xerosploit (like every other mitm scripts) and I choose a target (smartphone, notebook), the target will lose internet connection. How should I fix it?

Kali Linux 4.19.0-kali3-amd64 (dual boot)
Wifi: Alfa AWUS036NHA and AWUS036NH

**Comment by koeppea** [comment]
> Every time I use Xerosploit

What is that?

**Comment by Palmus84** [comment]
Sorry, I meant Ettercap.

**Comment by koeppea** [comment]
Can you please retry using LAN (wired) connection?

**Comment by Palmus84** [comment]
Yes, I could...but I need to make it work with wifi. Anyway I'll try wired!
